fiftyone/utils/transformers.py

```
@@ -1073,6 +1079,172 @@ def __init__(self, config):
+            a list of :class:`fiftyone.core.labels.Keypoints`
+        """
+        if self.preprocess and self._transforms is not None:
+            # Apply any preprocessing transforms
+            imgs = [self._transforms(img) for img in imgs]
+        
+        results = []
+        
+        for img in imgs:
+            # Ensure PIL Image
+            if torch.is_tensor(img):
+                img = F.to_pil_image(img)
```

Ruff (0.12.2)

Undefined name `F`

(F821)